tests/Postman/Mage2 Framework.postman_collection.json

```diff
@@ -0,0 +1,430 @@
+{
+	"info": {
+		"_postman_id": "3a6c8e83-627b-4042-97c4-7a0a3c383019",
+		"name": "Mage2 Framework",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "check_cart",
+			"description": null,
+			"item": [
+				{
+					"name": "check_cart",
+					"description": null,
+					"item": [
+						{
+							"name": "Check_cart android",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"id": "a6dcc1f0-89d1-44aa-9655-4fc29bffdf61",
+										"type": "text/javascript",
+										"exec": [
+											"pm.test(\"Cart has coupon\", function () {",
+											"    var jsonData = pm.response.json();",
+											"    pm.expect(jsonData.external_coupons).not.to.be.empty;",
+											"});"
+										]
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Content-Type",
+										"value": "*/*"
+									},
+									{
+										"key": "X-Shopgate-Auth-User",
+										"value": "{{sgHeaderAuthUser}}"
+									},
+									{
+										"key": "X-Shopgate-Auth-Token",
+										"value": "{{sgHeaderAuthToken}}"
+									}
+								],
+								"body": {
+									"mode": "formdata",
+									"formdata": [
+										{
+											"key": "action",
+											"value": "check_cart",
+											"type": "text"
+										},
+										{
+											"key": "shop_number",
+											"value": "{{shop_number}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][item_id]",
+											"value": "{{simple_item_id}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][item_number]",
+											"value": "{{simple_item_number}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][item_number_public]",
+											"value": "{{simple_item_number_public}}",
+											"type": "text"
+										},
+										{
+											"key": "cart[items][0][internal_order_info]",
+											"value": "{{simple_item_internal_order_info}}",
```

Comment:
this info may need to be static or the variables will need to be exported, it's not clear what this should be

Reply:
no no, this needs to be automated later. It needs to be committed as a separate json file to run via `newman`